### PR TITLE
Add info on plugin folders

### PR DESCRIPTION
**Pull request by Isengo1989**

### CODE REVIEW

File: guides/plugins/plugins/index.md [hunk 1]
```diff
@@ -8,3 +8,13 @@ nav:
...
+
+
+## Types of plugins
+There are different types of plugins in terms of their folder structure and functionality.
+
+### Plugins
+SHOPWARE_ROOT/custom/plugins contains all plugins from the shopware store. The plugins are installed and managed via the Shopware administration.
+
+### Static plugins
+SHOPWARE_ROOT/custom/static-plugins contains all plugins that are project-specific and are typically committed to the git repository. Static plugins are plugins that are not installed via the Shopware administration.
```
[the remark this answers is not in the document]
Comment (Isengo1989) [reply guides/plugins/plugins/index.md]: Thx I added the command and adjusted the text

image

File: guides/plugins/plugins/index.md [hunk 2]
```diff
@@ -8,3 +8,21 @@ nav:
...
+
+### Static plugins
+`<shopware project root>/custom/static-plugins` contains all plugins that are project-specific and are typically committed to the git repository.
+
+:::Info
+The detection of static plugins is not done via the Shopware administration. They have to be required by the project via composer to be installable.
+:::
+
+```bash
+composer req <vendor>/<plugin-name>
```
Comment (shyim) [inline guides/plugins/plugins/index.md:27]: maybe just a comment above with it's the `name` property of the composer.json of the plugin?

Comment (Isengo1989) [reply guides/plugins/plugins/index.md:28]: Done

**Comment by Isengo1989** [comment]
@shyim anything else? If not, please approve :slightly_smiling_face:

**Review by shyim — APPROVED** [review]

**Pull request merged**